packages/playground-server/src/middleware/matchmake.ts

```diff
@@ -0,0 +1,59 @@
+import { Context } from "koa";
+import "koa-body"; // See: https://github.com/dlau/koa-body/issues/109
+
+import { createErrorResponse, createErrorResponseForDatabase } from "../api";
+import { matchmakeUser } from "../db";
+import {
+  ApiResponse,
+  ErrorCode,
+  MatchmakeRequest,
+  MatchmakeResponseData,
+  PlaygroundUserData
+} from "../types";
+
+function validateRequest(params: MatchmakeRequest): ApiResponse {
+  if (!params.userAddress) {
```

Comment:
this shouldn't be necessary when this becomes an authenticated request

Reply:
Will be cleared on the Login route PR.